db/migrate/20161211042811_add_timestamps_to_offices.rb

```diff
@@ -1,7 +1,9 @@
+# rubocop:disable Rails/BulkChangeTable
 class AddTimestampsToOffices < ActiveRecord::Migration[4.2]
   def change
     add_column :offices, :created_at, :datetime
```

Comment:
not too familiar with rails, but it seems like these can be combined into one somehow?

Reply:
Do you mean adding something to the `.rubocop.yml`?

Reply:
never mind, I think this is fine, unless we can ignore the directory somehow

Reply:
I'm ignoring existing migrations so we don't modify them and keeping the rule open for any new migrations